code/modules/mob/living/carbon/carbon.dm

```diff
@@ -774,11 +774,14 @@ var/list/ventcrawl_machinery = list(/obj/machinery/atmospherics/unary/vent_pump,
 	spawn(0)
 		resist_muzzle()
 	if(restrained())
+		var/obj/item/I = handcuffed
+		var/breakouttime = I.breakouttime
+		var/displaytime = breakouttime / 600
 		changeNext_move(CLICK_CD_BREAKOUT)
 		last_special = world.time + CLICK_CD_BREAKOUT
 		visible_message("<span class='warning'>[src] attempts to unbuckle [p_them()]self!</span>", \
-					"<span class='notice'>You attempt to unbuckle yourself... (This will take around one minute and you need to stay still.)</span>")
-		if(do_after(src, 600, 0, target = src))
+					"<span class='notice'>You attempt to unbuckle yourself... (This will take around [displaytime] minute and you need to stay still.)</span>")
```

Comment:
For values of breakouttime other than 600 (1 minute) this will display weird things like `0.75 minute`, `0 minute`, or similar

Reply:
Right let me see if I can convert that to seconds.

Reply:
Got this wrapped up. Display time is in seconds, and changed uncuff time to follow suit.